scripts/ask_adult.sh

```diff
@@ -0,0 +1,22 @@
+# код языка
+#!/bin/bash 
+# вывод в командную строку сообщения
+echo -n "Введите значение переменной a  " 
+# запись ввода из командной строки в переменную а
+read a
+#оператор if  проверяет число ил нет
+if [ -n "$a" ] && [ "$a" -eq "$a" ] 2>/dev/null; then
+# оператор if и его условие
+if [ "$a" -ge 18 ] 
```

Comment:
You have to use code styling `https://ru.wikipedia.org/wiki/%D0%A1%D1%82%D0%B0%D0%BD%D0%B4%D0%B0%D1%80%D1%82_%D0%BE%D1%84%D0%BE%D1%80%D0%BC%D0%BB%D0%B5%D0%BD%D0%B8%D1%8F_%D0%BA%D0%BE%D0%B4%D0%B0`

Reply:
hello

Reply:
made changes